### PR TITLE
ENH(merge?)adding display code by jonas

### LINKED ISSUE #282: Visualization of Scattering Coefficients
Hi,

First I would like to thank you for the work you have done on Kymatio, really helpful 👍

I would like to visualize scattering coefficients like that for example (first and second orders):
capture

Do you know if there is an easy way to do that with Kymatio Scattering2D output?

Thank you!

### PR DESCRIPTION
**Pull request by edouardoyallon**
Close #282 

- [x] radii of different scales should sampled in a dyadic manner
- [x] seems the figures need to be rotated by pi/8
- [x] story lines should be improved
- [x] incorporating sphinx gallery compatibility

### CODE REVIEW

File: examples/2d/scattering_disk_display.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import matplotlib as mpl
+import matplotlib.cm as cm
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from kymatio import Scattering2D
+from PIL import Image
+
+
+def main():
```
Comment (eickenberg) [inline examples/2d/scattering_disk_display.py]: why would you hide all the information in a function? This is a script - I don't see a reason not to write all the commands in the outer scope

File: examples/2d/scattering_disk_display.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    plt.subplots_adjust(left=None, bottom=None, right=None, top=None, wspace=0.05, hspace=0.05)
+    offset = 0.1  # white circle in the middle from Bruna and Mallat paper...
+    for row in range(window_rows):
+        for column in range(window_columns):
+            ax = plt.subplot(window_rows, window_columns, 1 + column + row * window_rows, projection='polar')
+            ax.set_ylim(0, 1)
+            ax.axis('off')
+            ax.set_yticklabels([])  # turn off radial tick labels (yticks)
+            ax.set_xticklabels([])  # turn off degrees
+            # ax.set_theta_zero_location('N')  # 0° to North
```
Comment (eickenberg) [inline examples/2d/scattering_disk_display.py]: uncomment or delete

**Comment by edouardoyallon** [comment]
If someone is brave enough to provide me a digit from MNIST, as well as the corresponding screenshot from MATLAB(which I don't have), I finalize this PR.

**Comment by Jonas1312** [comment]
> 
> 
> If someone is brave enough to provide me a digit from MNIST, as well as the corresponding screenshot from MATLAB(which I don't have), I finalize this PR.

Digit: https://drive.google.com/file/d/1qXbZFjsoVoGMMKo3VZNlCUbI-b3qh7no/view?usp=sharing
ScatNet: https://github.com/kymatio/kymatio/issues/282#issuecomment-457900591

**Comment by edouardoyallon** [comment]
This is the corresponding figure.
sphx_glr_plot_scattering_disk_001

I guess the content can be merged!

**Comment by Jonas1312** [comment]
Thanks!

**Comment by eickenberg** [comment]
Thanks for this awesome example @Jonas1312

**Pull request merged**